ExampleApp/package.json

```diff
@@ -17,18 +17,23 @@
   "devDependencies": {
     "@optimove-inc/cordova-sdk": "file:../cordova-sdk",
     "cordova-android": "^11.0.0",
-    "cordova-ios": "^6.2.0",
+    "cordova-ios": "^7.1.1",
+    "ios": "^0.0.1",
```

Comment:
what's this one?

Reply:
also semver is not used?

Reply:
this will have been from removing and adding the platfom as part of development.  works with the latest version I will test with 6.2.0 as well.  semver is a dependacy pulled in from the cordova package and its dependacies

Reply:
smth seems not right, what is this package https://www.npmjs.com/package/ios
- published last time 10 y ago
- version 0.0.1
- keywords "hello world"
- doesnt have any description

to me seems like a random test package with xiaosheng trying out stuff, but having 1000s of downloads because of name "ios" :D

transitive dependencies shouldnt be added here by cordova

Reply:
some other packages he has

![image](https://github.com/user-attachments/assets/9b0e11d5-33b6-4026-b0e6-77fed183be3e)